web/src/server/auth.ts

```diff
@@ -264,6 +280,26 @@ const extendedPrismaAdapter: Adapter = {
 
     return user;
   },
+
+  async linkAccount(data) {
+    if (!prismaAdapter.linkAccount)
```

Comment:
why would link account not be implemented? I'd prefer not to throw an exception based on the availability of a method in a library.

Reply:
> why would link account not be implemented? I'd prefer not to throw an exception based on the availability of a method in a library.

<img width="1066" alt="Screenshot 2567-08-08 at 23:46:24" src="https://github.com/user-attachments/assets/88d4ea7e-24df-40dd-822c-177d725cf4ce">

This logic handle a type error since the authentication provider cannot determine it until it renders. My Lint keeps giving errors about calling an undefined function. However, if you have a suggestion for handling the case, feel free to suggest it to me.